src/tests/functional/shared_test_classes/src/base/utils/generate_inputs.cpp

```diff
@@ -220,10 +220,8 @@ ov::runtime::Tensor generate(const std::shared_ptr<ngraph::op::v0::FakeQuantize>
         case 4:
             return ov::test::utils::create_tensor<float>(elemType, targetShape, outputHighData, outputHighData.size());
         default: {
-            float min = +5.f, max = +25.f;
-
             InputGenerateData inGenData;
```

Comment:
Could you please check all fake quantize values? I am not sure that all used values are correct...

Reply:
@iefode Here change the range value to 10, according to the old workflow, which using the default value at here:
https://github.com/openvinotoolkit/openvino/blob/61c5a80fa63ed3dd7eb353c62ceee908ef9d8d69/src/tests/test_utils/functional_test_utils/include/functional_test_utils/blob_utils.hpp#L504-L508
Another reason is without this change, the following test about "*QuantizedConvolutionBatchNorm*" will failed, such as
![image](https://github.com/openvinotoolkit/openvino/assets/52686861/3e44207b-355c-4307-b64f-b11044875ea4)
I don't know how to check all fake quantize values. Can u share more details?

Reply:
Which details do you mean? It is parameters to generate input tensors for fake quantize.. I don't know fake quantize details implementation. It is the main reason why I have asked you

Reply:
Yes, this parameter is for fake quantize input generate. I compared the API 1.0 input data & API 2.0 input data, with this change can get the same input. And all tests related quantize can pass. So I think we need this change to keep the same as API 1.0. Here is a question, why for quantize operation the inputs need to be limited in 0~10? I'm not very familiar with quantize operation, anybody can help clarify this?

Reply:
@vurusovs can we merge this PR first? And create another ticket to track why for "BatchNormInference" operation the input range need to be limited to 0~10.

Reply:
Because inputs generation is shared between different types of tests, we need confirmation there is no impact  on other tests in terms of coverage
Waiting for @iefode and @eshoguli for confirmation

Reply:
@zhaixuejun1993 
I will discuss the ranges with @eshoguli as a  `fakeQuantize` expert.
Will provide the technical update in the PR comment.
Please do not merge before update

Reply:
> @zhaixuejun1993 I will discuss the ranges with @eshoguli as a `fakeQuantize` expert. Will provide the technical update in the PR comment. Please do not merge before update

That will be great! I will also do some learning about fakeQuantize

Reply:
@zhaixuejun1993, Ideally we should define data interval for `FakeQuantize` based on input intervals (2nd and 3rd inputs). If we have few failed tests then we can use static variables to use previous intervals to fix tests. If we have a lot of failed tests, then we can update intervals later. Thanks!

Reply:
@eshoguli Only the tests about "QuantizedConvolutionBatchNorm" failed. Maybe we can use previous intervals to fix this tests.

Reply:
@zhaixuejun1993 
We had a discussion with @eshoguli as LPT transformation owner. As he said, we will ask you to fix the failed tests in case it is a couple of tests to merge PR. In other case: our team will update the intervals for `fakeQuantize` as a part of LPT OV2.0 migration process. Thanks!